src/main/java/org/scijava/ui/DefaultUIService.java

```diff
@@ -257,16 +258,31 @@ public UserInterface getUI(final String name) {
 
 	@Override
 	public void show(final Object o) {
-		getDefaultUI().show(o);
+		final UserInterface defaultUI = getDefaultUI();
+		if (defaultUI == null) {
```

Comment:
Please use the more compact one-line syntax:
```java
if (defaultUI == null) return;
```
for consistency with the other blocks of code in these here parts.